Notifications can be sent in case of terminated Relationships (#286)
nmshd/documentation: _docs_integrate/delete-attributes.md

Then, a [Notification]({% link _docs_integrate/data-model-overview.md %}#notification) with a [PeerSharedAttributeDeletedByPeerNotificationItem]({% link _docs_integrate/data-model-overview.md %}#peersharedattributedeletedbypeernotificationitem) is generated and sent to the owner of the peer shared Attribute, informing them that you deleted the Attribute they shared with you.
Consequently, the `deletionInfo` of their corresponding own shared Attribute and of all potential predecessors is updated with `deletionStatus` `"DeletedByPeer"` and the time of receiving the Notification as `deletionDate`.
In case the owner already [deleted their own shared Attribute](#delete-own-shared-attributes), nothing happens.
Please further note that the Notification is queued if the [Relationship is currently terminated]({% link _docs_integrate/terminate-relationships.md %}#terminate-an-active-relationship).

> It can then only be received and processed once the [Relationship has been reactivated]({% link _docs_integrate/terminate-relationships.md %}#reactivate-a-terminated-relationship).

Reviewer: I'd explicitly state that's only the case if the Relationship is terminated and not yet decomposed, since an integrator might not be familiar with the terminology of Relationship termination in detail. Also, in the second sentence we could be more clear that reactivating a Relationship is not something that will necessarily happen, e.g. by using the conditional "if" instead of "once".

Apart from that, the same will hold true if the Relationship is not terminated, but the peer is in the grace period of an IdentityDeletionProcess. It would be great if we included that here.

Same for other occurrences.

Then, a [Notification]({% link _docs_integrate/data-model-overview.md %}#notification) with a [PeerSharedAttributeDeletedByPeerNotificationItem]({% link _docs_integrate/data-model-overview.md %}#peersharedattributedeletedbypeernotificationitem) is generated and sent to the owner of the peer shared Attribute, informing them that you deleted the Attribute they shared with you.
Consequently, the `deletionInfo` of their corresponding own shared Attribute and of all potential predecessors is updated with `deletionStatus` `"DeletedByPeer"` and the time of receiving the Notification as `deletionDate`.
In case the owner already [deleted their own shared Attribute](#delete-own-shared-attributes), nothing happens.

> Please further note that the Notification is queued if the [Relationship is currently terminated]({% link _docs_integrate/terminate-relationships.md %}#terminate-an-active-relationship) but not yet decomposed.

Reviewer: If you find it fitting, you can add a link to `terminate-relationships#decompose-a-relationship`.

> It can then only be received and processed if the [Relationship has been reactivated]({% link _docs_integrate/terminate-relationships.md %}#reactivate-a-terminated-relationship).

Reviewer: ```suggestion
It can then only be received and processed if the [Relationship is reactivated]({% link _docs_integrate/terminate-relationships.md %}#reactivate-a-terminated-relationship).
```

The Notification is also queued if the [peer is currently in deletion]({% link _docs_integrate/delete-identities.md %}#effects-of-identity-deletion-on-relationships) but not yet deleted.

> It can then only be received and processed if the peer has [cancelled its deletion]({% link _docs_use-cases/use-case-transport-cancel-identitydeletionprocess.md %}).

Reviewer: ```suggestion
It can then only be received and processed if the peer [cancels its deletion]({% link _docs_use-cases/use-case-transport-cancel-identitydeletionprocess.md %}).
```
